docs/hooli-partner-demo.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: hooli-partner-demo
+title: Hooli Partner Demo
+sidebar_label: Hooli Partner Demo
+slug: /hooli-partner-demo
+---
+
+import Tip from '../src/components/Tip';
+
+The open source Hooli partner demo demonstrates an example implementation of the issuing partner Verified Inc. digital identity cards which then redirects to Kredita. It is an application for a fictional customer, Hooli, which uses Verified Inc. to issue reusable <Tip type="credential">credentials</Tip> to its users and send them to the partner url.
+
+Additional information on how to issue credentials with a partner can be found in the Guide's issuance with partner [documentation](/issuance-guide#partner-issuance).
```

Comment:
```suggestion
Additional information on how to issue credentials with a partner can be found in the Partner Issuance [guide](/issuance-guide#partner-issuance).```

Reply:
This was resolved in https://github.com/UnumID/UnumID.github.io/pull/102